Mapper/BaseGroupedMapper.php

```diff
@@ -256,6 +262,16 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * TODO: make this method abstract before releasing 4.0
+     *
+     * @return string
+     */
+    /*abstract*/ protected function getName()
+    {
+        return 'default';
```

Comment:
You have to raise a deprecation warning here saying that getName() should be implemented if you're going to make this abstract.